cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,46 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = 'article';
+
+  get titleOfArticle() {
+    return cy.get('h1');
+  }
+
+  get editButton() {
+    return cy.get('.container > .article-meta > ' +
+    ':nth-child(3) > .btn-outline-secondary');
```

Comment:
The CSS selector for the `editButton` is quite specific and relies on the exact structure of the HTML. If the HTML structure changes, this selector might break. Consider using a more robust selector, such as a data attribute, if available.